install/update_91_92.php

```diff
@@ -53,67 +53,43 @@ function update91to92() {
    $backup_tables = false;
    // table already exist but deleted during the migration
    // not table created during the migration
-   $newtables     = array(
-      'glpi_businesscriticities',
-      'glpi_knowbaseitems_items',
-      'glpi_knowbaseitems_revisions',
-      'glpi_knowbaseitems_comments',
-      'glpi_devicecasemodels',
-      'glpi_devicecontrolmodels',
-      'glpi_devicedrivemodels',
-      'glpi_devicegraphiccardmodels',
-      'glpi_deviceharddrivemodels',
-      'glpi_devicememorymodels',
-      'glpi_devicemotherboardmodels',
-      'glpi_devicenetworkcardmodels',
-      'glpi_devicepcimodels',
-      'glpi_devicepowersupplymodels',
-      'glpi_deviceprocessormodels',
-      'glpi_devicesoundcardmodels',
-      'glpi_devicegenericmodels',
-      'glpi_devicegenerics',
-      'glpi_items_devicegenerics',
-      'glpi_devicegenerictypes',
-      'glpi_devicebatteries',
-      'glpi_items_devicebatteries',
-      'glpi_devicebatterytypes',
-      'glpi_devicefirmwares',
-      'glpi_items_devicefirmwares',
-      'glpi_devicefirmwaretypes',
-      'glpi_savedsearches_alerts',
-      'glpi_items_operatingsystems',
-      'glpi_operatingsystemkernels',
-      'glpi_operatingsystemkernelversions',
-      'glpi_operatingsystemeditions'
-   );
+   $newtables     = array("glpi_bookmarks",
+                          "glpi_bookmarks_users",
+                          "glpi_queuedmails");
 
```

Comment:
Yes, only tables created in the update script should be in the array; they'll be renamed as backup on further runs.